internal/codegen/tpl_file.go

```diff
@@ -170,22 +170,60 @@ func (f *TplFile) Path() string {
 //	{{- file.Create (printf "cmd/%s.go" $commandName) 0600 now }}
 //	{{- stencil.Include "command" | file.SetContents }}
 //	{{- end }}
-func (f *TplFile) Create(path string, mode os.FileMode, modTime time.Time) (out, err error) {
+func (f *TplFile) Create(path string, mode os.FileMode, modTime time.Time) (out string, err error) {
```

Comment:
Good catch, kinda hilarious this hadn't been caught before. I'm assuming it's because we usually only call `file.Create` in a "virtual file" template.

Reply:
ya i assumed that too